src/react/account/AccountUserSecretKeyModal.tsx

```diff
@@ -130,8 +140,17 @@ const modalBody = (key: AccountKey | null) => {
           </T.Row>
         </T.Body>
       </Table>
+      <Box
+        display={'flex'}
+        style={{ alignItems: 'end', flexDirection: 'column' }}
+      >
+        <StyledCopybutton
+          text={`${key.accountName} ${key.accessKey} ${key.secretKey}`}
+          labelName="to Clipboard"
+        />
```

Comment:
the format suggested in the ticket should be:
```suggestion
        <StyledCopybutton
          text={`Account Name\t${key.accountName}\nUsername\t${IAMUserName}\nAccess key ID\t${key.accessKey}\nSecret Access key\t${key.secretKey}\n`}
          labelName="to Clipboard"
        />
```